### ISSUE TITLE
A bug in Version 0.10.13:  It should be "i" in ellist not "i ".

### ISSUE THREAD

**Issue by liuguanfu1120** [AtomDB/pyatomdb#46]
The original code is as follows
```python
def elsymb_to_Z(elsymb) :
  """
  Converts element symbol to nuclear charge, e.g. "C" -> 6

  Parameters
  ----------

  elsymb : str
    Element symbol, e.g. "C". Case insensitive.

  Returns
  -------
  int
    Z for the ion. (e.g. 6 for C)
  """
#
#  Version 0.1 28 July 2009
#  Adam Foster
#
  ellist=('h' , 'he', 'li', 'be', 'b' , 'c' , 'n' , 'o' , 'f' , 'ne',
          'na', 'mg', 'al', 'si', 'p' , 's' , 'cl', 'ar', 'k' , 'ca',
          'sc', 'ti', 'v' , 'cr', 'mn', 'fe', 'co', 'ni', 'cu', 'zn',
          'ga', 'ge', 'as', 'se', 'br', 'kr', 'rb', 'sr', 'y' , 'zr',
          'nb', 'mo', 'tc', 'ru', 'rh', 'pd', 'ag', 'cd', 'in', 'sn',
          'sb', 'te', 'i ', 'xe', 'cs', 'ba', 'la', 'ce', 'pr', 'nd',
          'pm', 'sm', 'eu', 'gd', 'tb', 'dy', 'ho', 'er', 'tm', 'yb',
          'lu', 'hf', 'ta', 'w' , 're', 'os', 'ir', 'pt', 'au', 'hg',
          'tl', 'pb', 'bi', 'po', 'at', 'rn', 'fr', 'ra', 'ac', 'th',
          'pa', 'u')

  try:
    ind=ellist.index(elsymb.lower().strip())
  except ValueError:
    print("elsymb_to_z0 error: invalid element symbol '"+elsymb+"', returning -1")
    ind=-1

  return ind+1
```
However, there is a space after "i" (the 53th element Iodine). Therefore, if you try
```python
import pyatomdb
print(pyatomdb.atomic.Ztoelsymb(53))
``` 
you will get 
```python
"I "
```
it looks good. However, if you try
```python
pyatomdb.atomic.elsymb_to_Z("I ")
```
an error appears
```python
elsymb_to_z0 error: invalid element symbol 'I ', returning -1
```
The bug is obvious:
```
ind=ellist.index(elsymb.lower().strip())
```
".strip" will strip the space after "I", but there is no "i" in ellist but "i ".
In conclusion, I think "i " in ellist should be replaced by "i".

**Comment by jagophile** [comment]
Thanks - that's definitely a bug. Will fix it now.

**Comment by jagophile** [comment]
I believe this is all fixed now and uploaded as 0.11.1.

Please let me know if you have any more issues, and thank you again.

Cheers,
Adam
